### ISSUE TITLE
Store fonts in storage rather than in the vendors folder and allow fonts path to be set in a config file.

### ISSUE THREAD

**Issue by Gnative** [Dolphiq/craft3-iconpicker#8]
Storing the fonts within the vendors directory doesn't seem right to me, this should be a folder that we shouldn't touch.

We should also be able to set the fonts folder with in a config file so as to be able to suit our projects structure.

**Comment by 24hoursmedia-craftcms** [comment]
yes please, I tried to deploy on production using git pull and composer install etc, but there is no sound way to get the fonts into the vendor directory. One should not write in the vendor directory anyways.

**Comment by bertoost** [comment]
By default it should be in the `storage/` folder somewhere and configurable if you already have FA included for the frontend website (or something like that)

**Comment by devotoare** [comment]
This. Having this not be the default (which it should be) or even configurable at the very least stops me from using this package completely. Too much messing around with gitignores and stuff to make this work across environments.

**Comment by mateostabio** [comment]
Any solutions on this? Does anyone have instructions on how to put my TTF files in my /web/assets/ folder?
My live server is not displaying the icons and bugging out my admin.

**Comment by cosly** [comment]
+1

**Comment by EternalBlack** [comment]
Seems like [this](https://github.com/Dolphiq/craft3-iconpicker/blob/master/src/fields/Iconpicker.php#L31) is the corresponding line of code causing us all trouble.

I'll fork the repo and try to fix this issue and'll open a pull-request afterwards.

**Comment by kisonay** [comment]
Any update on this?

**Comment by devotoare** [comment]
> Any update on this?

Nope this project seems abandoned. We moved on years ago.
